feature/helmchart: supported private helm chart repo to install dapr cluster (#2671)
dapr/docs: daprdocs/content/en/reference/environment/_index.md

| DAPR_API_TOKEN  | Dapr sidecar     | The token used for Dapr API authentication for requests from the application. [Enable API token authentication in Dapr]({{< ref api-token >}}). |
| NAMESPACE | Dapr sidecar | Used to specify a component's [namespace in self-hosted mode]({{< ref component-scopes >}}). |
| DAPR_DEFAULT_IMAGE_REGISTRY | Dapr CLI | In self-hosted mode, it is used to specify the default container registry to pull images from. When its value is set to `GHCR` or `ghcr`, it pulls the required images from Github container registry. To default to Docker hub, unset this environment variable. |
| SSL_CERT_DIR | Dapr sidecar | Specifies the location where the public certificates for all the trusted certificate authorities (CA) are located. Not applicable when the sidecar is running as a process in self-hosted mode.|

> | HELM_CHART_REPO_URL | Your private dapr helm chart url  | Specifies a private dapr helm chart url, defaultly Official Helm Chart: `https://dapr.github.io/helm-charts`|

Reviewer: ```suggestion
| HELM_CHART_REPO_URL | Your private Dapr Helm chart url  | Specifies a private Dapr Helm chart url, which defaults to the official Helm chart URL: `https://dapr.github.io/helm-charts`|
```

> | HELM_CHART_REPO_USERNAME | A username for a private helm chart | The username required to access a private dapr helm chart. If it can be accessed publicly, the env variable does not need to be set| 

Reviewer: ```suggestion
| HELM_CHART_REPO_USERNAME | A username for a private Helm chart | The username required to access the private Dapr Helm chart. If it can be accessed publicly, this env variable does not need to be set| 
```

> | HELM_CHART_REPO_PASSWORD | A password for a private helm chart  |The password required to access a private dapr helm chart. If it can be accessed publicly, the env variable does not need to be set| |

Reviewer: ```suggestion
| HELM_CHART_REPO_PASSWORD | A password for a private Helm chart  |The password required to access the private Dapr helm chart. If it can be accessed publicly, this env variable does not need to be set| |
```

Reviewer: @1046102779 - You should also update this article with a section https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/#install-with-helm-advanced on how to use these helm chart environment variables. This makes it more discoverable and you can provide examples on how to use this. Please add to this best practice document

Author: Thank you so mush, msfussell.

I just submitted another version according to your comments. I need you to help me.
